test/config/validation.spec.js

```diff
@@ -18,7 +18,7 @@ describe('config/validation', () => {
         config
       );
       expect(warnings).toHaveLength(0);
-      expect(errors).toHaveLength(2);
+      expect(errors).toHaveLength(4);
```

Comment:
Let's fix the test by putting the selectors inside packageRules so that the error count remains at 2